docs/modules/ROOT/pages/tmail-backend/jmap-extensions/calendarEventReply.adoc

```diff
@@ -195,3 +201,74 @@ If the number of blob ids in the request exceeds the limit (currently 16), the s
     "c1"
 ]
 ----
+
+== CalendarEventCounter/accept
+
+This method allows the client to accept a calendar event counter from a participant.
+The CalendarEventCounter/accept method takes the following arguments:
+
+- *accountId*: `Id` The id of the account to use.
+- *blobIds*: `Id[]` The ids correspond to the blob of calendar event counter files that the user intends to accept.
+They must be validated to ensure they can be parsed using the `CalendarEvent/parse` method for verification.
+Note that contrary to `CalendarEvent/accept`, here we are parsing an .ics file with a `METHOD:COUNTER` field, and not `METHOD:REQUEST`.
+- *language*: `String`. Optional, Default is `en`. The language tag that the client wants the server to use when determining the email template.
+
+The response object contains the following arguments:
+
+- *accountId*: `Id` The id of the account used for the call.
+- *accepted*: `Id[CalendarEvent[]]|null` A list of ids of the calendar event counters that were successfully accepted, or `null` if none.
+- *notFound*: `Id[]|null` A list of blob ids given that could not be found, or `null` if none.
+- *notAccepted*: `Id[SetError]|null`  A map of the blobId to a SetError object for each calendar event counter that failed to reply, or null if all successful.
+
+Note that the `counterSupport` property defined with the capability `com:linagora:params:calendar:event` must be true for this method to be available.
+
+=== Example
+
+The client makes a request to accept a calendar event counter:
+
+....
+{
+	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:calendar:event"],
+	"methodCalls": [
+        [ "CalendarEventCounter/accept", {
+            "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+            "blobIds": ["0f9f65ab-dc7b-4146-850f-7f59921a4a76"]
+        }, "c1"]
+	]
+}
+....
+
+The server responds:
+
+```
+[[ "CalendarEventCounter/accept",
+{
+    "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+    "accepted": [ "0f9f65ab-dc7b-4146-850f-7f59921a4a76" ],
+}, "c1" ]]
+```
+
+In the case that a blob id is not found, the server would respond:
+
+```
+[[ "CalendarEventCounter/accept",
+{
+    "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+    "notFound": [ "0f9f65ab-dc7b-4146-850f-7f59921a4a76" ]
+}, "c1" ]]
+```
+
+If the blob id has been found but can't parse or reply to the email, the server would respond:
+
+```
+[[ "CalendarEventCounter/accept",
+{
+    "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+    "notAccepted": {
+        "0f9f65ab-dc7b-4146-850f-7f59921a4a76": {
+            "type": "invalidPatch",
+            "description": "The calendar must have COUNTER as a method"
+        }
+    }
+}, "c1" ]]
+```
```

Comment:
Maybe we could document trying to call `CalendarEventCounter/accept` when it is not supported?

(InvalidArguments or unknownMethod)

Reply:
Fair